pyvista/plotting/mapper.py

```diff
@@ -123,10 +122,10 @@ def set_scalars(
                         raise TypeError('component must be either None or an integer')
                     if component is None:
                         scalars = np.linalg.norm(scalars.copy(), axis=1)
-                        title = '{}-normed'.format(title)
+                        scalars_name = '{}-normed'.format(scalars_name)
                     elif component < scalars.shape[1] and component >= 0:
                         scalars = scalars[:, component].copy()
-                        title = '{}-{}'.format(title, component)
+                        scalars_name = '{}-{}'.format(scalars_name, component)
```

Comment:
Since we already have f-strings earlier we should consider making these also f-strings. I would personally find them more readable that way.